pyrasa/utils/peak_utils.py

```diff
@@ -1,22 +1,24 @@
 """Utilities for extracting peak parameters."""
 
+from collections.abc import Iterable
+
 import numpy as np
 import pandas as pd
 import scipy.signal as dsp
 
 
 # %% find peaks irasa style
 def get_peak_params(
-    periodic_spectrum,
-    freqs,
-    ch_names=[],
-    smooth=True,
-    smoothing_window=1,
-    polyorder=1,
-    cut_spectrum=(1, 40),
-    peak_threshold=1,
-    min_peak_height=0.01,
-    peak_width_limits=(0.5, 12),
+    periodic_spectrum: np.ndarray,
+    freqs: np.ndarray,
+    ch_names: Iterable = [],
```

Comment:
Never use mutable types (lists, dicts, etc) as default arguments. Replace this by a tuple.